src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -43,14 +45,56 @@ export const SchedulePageModal: FC<SchedulePageModalProps> = ({
     Record<string, WithId<AnyVenue>>
   > = relatedVenues.reduce(itemsToObjectByIdReducer, {});
 
+  const [startDate, setStartDate] = useState(START_DATE);
+
+  const selectStartDate = () => (
+    <>
+      <form
+        onClick={(e) => {
+          e.stopPropagation();
+        }}
+      >
+        <label>
+          Select the calendar starting date:
+          <div>
+            <select
+              value={startDate}
+              onChange={(e) => {
+                setStartDate(e.target.value);
+              }}
+            >
+              <option value="today">Today</option>
+              <option value="course">Educational Course</option>
+              <option value="conference">Conference</option>
+            </select>
+          </div>
+        </label>
+      </form>
+    </>
+  );
```

Comment:
This should either be extracted into a sub-component, or just rendered inline in the JSX below. We try not to use this 'render function' pattern (and where it exists currently is legacy code).

You also don't need to wrap this in the fragment tags (`<>` `</>`) as you already have a single root element (`form`)

Reply:
The `onChange` value would need to be extracted into a semantically named `const` and `useCallback`'d, otherwise it will cause re-renders due to the inline function definition.